src/test/simple/lifeguard.sol

```diff
@@ -0,0 +1,84 @@
+pragma solidity >=0.4.24;
+
+contract TokenLike {
+    function transferFrom(address, address, uint) public;
+    function mint(address, uint) public;
+    function approve(address usr, uint wad) public returns (bool);
+}
+
+contract PileLike {
+    function want() public returns (int);
+}
+
+contract OperatorLike {
+    function provide(address,address,uint,uint) public;
+    function release(address,address,uint,uint) public;
+    function free(address, uint) public;
+}
+
+contract SimpleLifeguardFab {
+    function deploy(address pile_, address operator_, address lightswitch_) public returns (address) {
+        SimpleLifeguard lifeguard = new SimpleLifeguard(pile_, operator_);
+        lifeguard.rely(msg.sender);
+        return address(lifeguard);
+    }
+}
+
+// Tranche Manager
+contract SimpleLifeguard {
```

Comment:
why is it called `lifeguard`?

Reply:
are we calling the tranche manager lifeguard?

Reply:
we can also call it tranche manager ;) it was to go with surfer but if we are removing it then we can update.